src/extension/src/constants.ts

```diff
@@ -160,7 +160,24 @@ export const CONSTANTS = {
     SIGNALR_API_SYNC_METHOD_NAME: "SyncTemplates",
     SIGNALR_API_GENERATE_METHOD_NAME: "Generate",
     MAX_SYNC_REQUEST_ATTEMPTS: 51,
-    SYNC_RETRY_WAIT_TIME: 250
+    SYNC_RETRY_WAIT_TIME: 250,
+    Endpoints: {
```

Comment:
If it is constant, should it be ENDPOINTS?

Reply:
Good question! I copied it over from the [client constants](https://github.com/microsoft/WebTemplateStudio/blob/dev/src/client/src/services/constants.tsx) but I could update it to uppercase